### ISSUE TITLE
Missing 'Infinite Wireless Transmitter'

### ISSUE THREAD

**Issue by ddoyle2** [scalda/Foolcraft_3#327]
## Issue Description
I upgraded from 1.5.1 to 1.5.4 and when I logged in, my infinite wireless transmitter from Refined Storage was missing, and it wasn't available for replacement.

## Reproduce Steps
- Have one in 1.5.1
- Upgrade to 1.5.4
- Find that it is missing

**Comment by Revvilo** [comment]
This seems like the cause. I've been wondering where it went too... bit odd.

![image](https://user-images.githubusercontent.com/13816212/46651433-7a5e5600-cbfc-11e8-9885-63721765d49a.png)

**Comment by generrosity** [comment]
Its intentional. Its similar to how the author removed one of the crafting options for two versions while he made it more stable. Users freaked out threatening never to use the mod again, and now love the mod more.

tl;dr: Im sure it will eventually be back. Its a bit OP anyway 😜
